tests/strategies/test_dataresource.py

```diff
@@ -136,26 +165,41 @@ def test_fetch_fails(
         data_resource.fetch(session_id=123)
 
 
+@pytest.mark.parametrize("backend", ["services", "python"])
 def test_initialize(
+    backend: str,
     mock_ote_response: "OTEResponse",
     ids: "Callable[[Union[ResourceType, str]], str]",
     server_url: str,
 ) -> None:
     """Test `DataResource.fetch()`."""
     import json
 
-    from otelib.strategies.dataresource import DataResource
+    if backend == "services":
+        from otelib.backends.services.dataresource import DataResource
+    elif backend == "python":
+        from otelib.backends.python.dataresource import DataResource
+
+        server_url = "python"
+        from oteapi.plugins import load_strategies
+
+        load_strategies()
+        from otelib.backends.python.base import Cache
+
+        Cache().clear()  # Cleanup the cache from other tests
```

Comment:
I think this part, which is repeated over and over can be simplified, but I'll have to think more about it. Perhaps this could be part of a separate issue and PR? :)

Reply:
Agreed, let's make this a separate issue.